library/src/main/java/com/novoda/downloadmanager/lib/DownloadInfo.java

```diff
@@ -40,10 +40,27 @@ public Reader(ContentResolver resolver, Cursor cursor) {
             mCursor = cursor;
         }
 
-        public DownloadInfo newDownloadInfo(Context context, SystemFacade systemFacade, StorageManager storageManager, DownloadNotifier notifier) {
-            final DownloadInfo info = new DownloadInfo(context, systemFacade, storageManager, notifier);
+        public DownloadInfo newDownloadInfo(
+                Context context,
+                SystemFacade systemFacade,
+                StorageManager storageManager,
+                DownloadNotifier notifier,
+                DownloadClientReadyChecker downloadClientReadyChecker) {
+
+            Fuzz fuzz = new Fuzz();
```

Comment:
Now might be a good time to rename it to something like `RandomNumberGenerator`?